commit 33aba8eef68b1745d3de96b609ff8296b70d9a1c
Author: Paul Floyd <pjfloyd@wanadoo.fr>
Date:   Wed Oct 27 21:37:00 2021 +0200

    Bug 444495 - dhat/tests/copy fails on s390x
    
    Add -fno-builtin to ensure that the copy functions get called and so dhat
    can intercept and count them.

diff --git a/dhat/tests/Makefile.am b/dhat/tests/Makefile.am
index 86a9b6d64..b86fc416d 100644
--- a/dhat/tests/Makefile.am
+++ b/dhat/tests/Makefile.am
@@ -29,3 +29,6 @@ AM_CXXFLAGS += $(AM_FLAG_M3264_PRI)
 # We don't care about uninitialized or unused malloc results
 basic_CFLAGS  = $(AM_CFLAGS) -Wno-uninitialized
 big_CFLAGS  = $(AM_CFLAGS) -Wno-unused-result
+
+# Prevent the copying functions from being inlined
+copy_CFLAGS = $(AM_CFLAGS) -fno-builtin
